types/laravel-vapor/index.d.ts

```diff
@@ -20,6 +20,7 @@ interface VaporStoreOptions {
 
 declare class Vapor {
     store(file: File, options?: VaporStoreOptions): Promise<any>;
+    withBaseAssetUrl(url?: string | boolean): void;
```

Comment:
```suggestion
    withBaseAssetUrl(url?: string): void;
```

The only useful value would be a string and i think a boolean might actually cause an error when trying to concatenate it with a string.

Reply:
Works for me, usage in the docs would be `Vapor.withBaseAssetUrl(import.meta.env.VITE_VAPOR_ASSET_URL as string)` and if that's `'true'` then that's on the user.